content/en/docs/reference/glossary/identity.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Identity
+---
+
+An identity is a unique name backed by a key material, for
+example:
+
+- [X.509 certificate](https://en.wikipedia.org/wiki/X.509)
+- [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)
+
+Each identity Istio assigns is understood throughout the mesh and you can use it
+to enable mutual authentication and enforce policies.
+
+An identity contains a substring identifying the mesh that created it. For
```

Comment:
You're duplicating the definition of trust domain here.  We should probably remove.

Reply:
Or replace it with a link to the Trust Domain entry.